### PR TITLE
Updates for new locale

### PR DESCRIPTION
**Pull request by greg-hellings**
Updates to the role I imported to bring it up to date.

### CODE REVIEW

File: handlers/main.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 - name: restart docker
-  become: true
+  become: "{{ docker_become }}"
+  become_user: "{{ docker_become_user }}"
   service:
     name: docker
     service: restarted
```
Comment (andrew103) [inline handlers/main.yml:6]: Most, if not all, the oasis roles usually put the `become` and `become_user` options after the call to the module. Do we want to maintain that sort of format or is it not a topic of concern?

File: molecule/default/molecule.yml [hunk 2]
```diff
@@ -9,16 +11,23 @@ lint:
 platforms:
   - name: test-docker
     image: Fedora 29
-    flavor: m1.medium
+    flavor: ci.m1.small
+    groups:
+      - fedora
```
Comment (andrew103) [inline molecule/default/molecule.yml:16]: Shouldn't these be set in the local `clouds.yaml` file? The `molecule/<scenario>/create.yml` files also don't reflect usage of the `clouds.yaml` file.

**Review by tehsmyers — APPROVED** [review]
dawkah dawkah dawkah dawkah dawkah dawkah dawkah dawkah dawkah dawkah dawkah

**Comment by robled** [comment]
dockah dockah dockah

**Comment by tehsmyers** [comment]
Jenkins is showing failures, but they're related to ansible changing the python "api" again, and testinfra depending on it. This is fixed in testinfra 3, which switched to ansible-runner, and should be depended on by the next version of molecule. We could probably also force it, similar to the workaround[0] currently in place to deal with a weird dependency matrix issue, since no changes were needed to molecule to adapt to testinfra 3.

[0] https://github.com/oasis-origin/oasis-pipeline/commit/5f47727df7fa0947527406d11216c6570b8fcf0c

File: molecule/default/molecule.yml (hunk 2 above)
Comment (greg-hellings) [reply molecule/default/molecule.yml:16]: In this case, they are being overridden here. ci.m1.medium is the default, but I don't need all those resources so I'm opting for ci.m1.small here. Group definition is not in clouds.yaml at all.

Also, once we move to using the newer version of molecule that's soon to be released (right @seandst ?!), we will change to using the molecule_openstack_ci role which incorporates that. We aren't bothering to retrofit that into every create.yml playbook.

File: handlers/main.yml (hunk 1 above)
Comment (greg-hellings) [reply handlers/main.yml:6]: There is no such specification in the OASIS standards. Most of the roles I authored do not (I prefer the order to be name, become, become_user, <module>, args, then follow up with when/register/loop/until after that), many of the roles authored by others do.

Ultimately, I don't think it matters too much, although it's nice to at least keep it the same inside of each tasks file.

**Review by andrew103 — APPROVED** [review]

**Pull request merged**